### PR TITLE
Rdm 10264 enable auto-scale

### PR DESCRIPTION
**Pull request by nathan-clark**
DISABLED 203 FT

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by rafalkalita — APPROVED** [review]

**Review by smathangi — APPROVED** [review]

File: charts/aac-manage-case-assignment/values.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@ java:
   image: 'hmctspublic.azurecr.io/aac/manage-case-assignment:latest'
   ingressHost: aac-manage-case-assignment-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: aac
+  autoscaling:
+    enabled: true 
+    maxReplicas: 4
```
Comment (MSancaktutar) [inline charts/aac-manage-case-assignment/values.yaml:8]: pls let's also include minReplicas.
I suggest 2 for it.

File: src/functionalTest/resources/features/F-203 - Unassign Access Within Organisation/F-203 - Unassign Access Within Organisation.feature [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #=================================================
-@F-203
+@F-203 @Ignore
```
Comment (MSancaktutar) [inline src/functionalTest/resources/features/F-203 - Unassign Access Within Organisation/F-203 - Unassign Access Within Organisation.feature:2]: Could you create a new ticket to restore this back?

File: charts/aac-manage-case-assignment/values.yaml (hunk 1 above)
Comment (nathan-clark) [reply charts/aac-manage-case-assignment/values.yaml:8]: default is 2 already

File: src/functionalTest/resources/features/F-203 - Unassign Access Within Organisation/F-203 - Unassign Access Within Organisation.feature (hunk 2 above)
Comment (nathan-clark) [reply src/functionalTest/resources/features/F-203 - Unassign Access Within Organisation/F-203 - Unassign Access Within Organisation.feature:2]: https://tools.hmcts.net/jira/browse/RDM-10485

**Pull request merged**